modules/multi-architecture-scheduling-examples.adoc

```diff
@@ -0,0 +1,140 @@
+// module included in the following assembly 
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc
+
+:_content-type: CONCEPT
+[id="multi-architecture-scheduling-examples_{context}"]
+
+= Sample multi-architecture node workload deployments 
+
+There are various cases to consider when you are scheduling workloads on a cluster with compute nodes of different architectures. 
+
+Using node affinity to schedule workloads on a node:: To allow a workload to be scheduled on only a set of nodes with architectures supported by its images, you can set the `spec.affinity.nodeAffinity` field in your pod template specification.
++
+.Example deployment with the `nodeAffinity` set to certain architectures 
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values: <1>
+                - amd64
+                - arm64
+----
+<1> Specify the set of supported architecture. Valid values include `amd64` and `arm64`.
+
+Tainting nodes for a specific architecture:: You can taint a node to avoid workloads that are not compatible with its architecture to be scheduled on that node. In clusters using a `MachineSet` object, you can add parameters to a `.spec.template.spec.taints` field to avoid workloads being scheduled on nodes with non-supported architectures.
```

Comment:
I'm not sure of the sentence 
```
In clusters using a `MachineSet` object
```

I used to mention it this way to exclude clusters like the ones installed via UPI methods because they do not leverage the MachineSet. However, I do not know how we refer to these cases in the doc for other scenarios.

Reply:
Agree, lets reword to something like:

> Tainting every node for a specific architecture:: You can add a taint to a node to avoid workloads that are not compatible with its architecture.  In the case that your cluster can use a `MachineSet` object, you can add parameters to a ".spec.template.spec.taints" field to avoid workloads being scheduled on nodes with non-supported architectures.